7Cav-Alive.Altis/init/fn_serverPreInit.sqf

```diff
@@ -1,3 +1,28 @@
 if (not isServer && hasInterface) exitWith {};
 
+#include "\serverscripts\zeusserverscripts\secretKey.sqf" //Includes the zeus security key.sqf so we can reference it for zeus IDs
+#include "..\scripts\baseCleanup\clearItemsBASE_switch.sqf" //Base script
+#include "..\scripts\server\deleteWeaponHolder.sqf" //Includes the deleteWeaponHolder script so everything can reference it 
+
+missionNamespace setVariable["LOYALTY_CIPHER", CIPHERSECRETKEY, true]; //variable init for loyalty cipher
+missionNamespace setVariable["endMissionNumber",0]; //Variable init for missionEnd scripts located in scripts/missionEnd
+
+//Initial Task Creation for all strongpoints
+//Airfield
+private _endMission = [BLUFOR, "Deactivate the Airfield!" Term1, ["Deactivate the satellite to prevent the enemy air commander from being able to call in air strikes!" _target, "Defend The Satellite!", "_target"], Term1, true] call BIS_fnc_taskCreate; //create the task (google it, it's under taskTutorial)
```

Comment:
I think you're missing some commas here?
```suggestion
private _endMission = [BLUFOR, "Deactivate the Airfield!", Term1, ["Deactivate the satellite to prevent the enemy air commander from being able to call in air strikes!", _target, "Defend The Satellite!", "_target"], Term1, true] call BIS_fnc_taskCreate; //create the task (google it, it's under taskTutorial)
```

Reply:
Not resolved, youre missing commas here and on the other lines

Reply:
Additionally, I think you have the syntax for this method wrong.